README.md

```diff
@@ -24,13 +24,10 @@ Alternatively you can clone this repository with
 git clone https://github.com/probabilistic-numerics/probnum
```

Comment:
Not this line, but the one above between both installation.
I would change it to something like 
"Alternatively, you can install this package from source with"